<!DOCTYPE HTML>
<!--
	Read Only by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Dylan Taylor - Genomics @ JHU</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<link rel="icon" href="images/web-icon_avatar.png">
	</head>
	<body class="is-preload">

		<!-- Header -->
			<section id="header">
				<header>
					<span class="image avatar"><img src="images/avatar.png" alt="" /></span>
					<h1 id="logo"><a href="#">Dylan Taylor</a></h1>
					<p>Ph.D. Candidate<br />
					Department of Biology<br />
					Johns Hopkins University</p>
				</header>
				<nav id="nav">
					<ul>
						<li><a href="#about" class="active">About</a></li>
						<li><a href="#research">Research Projects</a></li>
						<li><a href="#awards">Honors and Awards</a></li>
						<li><a href="#teaching">Teaching</a></li>
						<li><a href="#service">Academic Service</a></li>
						<li><a href="#cv">Curriculum Vitae</a></li>
					</ul>
				</nav>
				<footer>
					<ul class="icons">
						<li><a href="https://twitter.com/tlrdln22" class="icon brands fa-twitter" target="_blank"><span class="label">Twitter</span></a></li>
						<li><a href="https://github.com/dtaylo95" class="icon brands fa-github" target="_blank"><span class="label">Github</span></a></li>
						<li><a id="mlink" href="#" class="icon solid fa-envelope"><span class="label">Email</span></a></li>
					</ul>
				</footer>
			</section>

		<!-- Wrapper -->
			<div id="wrapper">

				<!-- Main -->
					<div id="main">

						<!-- About -->
							<section id="about">
								<div class="image main" data-position="center">
									<img src="images/UTL.png" alt="" />
								</div>
								<div class="container">
									<header class="major">
										<!-- <h2>About Me</h2> -->
										<h3>Investigating functional genetic variation<br />
										in diverse human populations.</h3>
									</header>
									<p>I am a Ph.D. candidate in the <a href="https://cmdb.jhu.edu/" target="_blank">Cell, Molecular, Developmental Biology and Biophysics (CMDB)</a> program at Johns Hopkins University, advised by <b><a href = "https://mccoy-lab.org" target="_blank">Dr. Rajiv McCoy</a></b>.<br />
									<br />
									My research focuses on developing tools and datasets that facilitate deeper insight into human genetic variation. As a member of the <b><a href="https://sites.google.com/ucsc.edu/t2tworkinggroup" target="_blank">T2T Consortium</a></b>, I've explored the utility of a complete telomere-to-telomore reference genome in studying human genetic variation. My own work focuses on generating large, globally diverse human transcriptomics datasets, and developing computational and statistical methods that use these datasets to investigate genetic variation underlying gene expression and splicing differences between individuals. Through functional annotation of this variation, we can uncover the molecular mechanisms by which genetic variation drives human traits and disease.<br />
									<br />
									I obtained my B.S. in Biology from the University of Maryland, College Park. There, I worked with <b><a href="https://pop-lab.org/" target="_blank">Dr. Mihai Pop</a></b> to develop computational tools for high-resolution metagenomic profiling.</p>
								</div>
							</section>

						<!-- Research Projects -->
							<section id="research">
								<div class="container">
									<header class="major">
										<h2>Research Projects</h2>
									</header>
									<h4>Exploring human regulatory sequences</h4>
									<a id="MAGE_paper_link" target="_blank" class="image" style="width:80%"><img src="images/1KGP_map.png" alt="" style="width:100%"/></a>
									<p style="font-size:12px;width:60%;line-height:20px;">The MAGE resource comprises RNA-seq data from 731 individuals across 26 globally diverse populations. Adapted from <span id="MAGE_paper_shortRef" /></p>
									<p>The vast majority of human trait-associated genetic variation lies in non-coding regions of the genome. Additionally, genetic variation affecting gene expression and alternative splicing has been shown to be a major contributor of human traits and disease. Unfortunately, the molecular mechanisms by which trait-associated non-coding variation drive phenotypic differences between individuals are still not well-characterized.<br />
									<br />
									Previous studies investigating these links in humans have been strongly biased toward participants of European ancestries, diminishing generalizability and hindering functional and evolutionary research. To address these limitations, we developed <a id="MAGE_paper_link" target="_blank">MAGE</a>, a resource for Multi-ancestry Analysis of Gene Expression. MAGE is an open-access RNA-seq data set of lymphoblastoid cell lines from 731 individuals from the 1000 Genomes Project spread across 5 continental groups and 26 populations. We used MAGE to identify tens of thousands of independent expression- and splicing-linked genetic variants, many of which are private to previously under-represented populations. We also observe that "population-specific" effects are rare, and properlys-specified models using diverse data sets produce results that replicate between populations. We hope MAGE acts an inclusive resource for more equitably studying the evolution and function of human genomes. Check out the <a href="https://github.com/mccoy-lab/MAGE" target="_blank">GitHub page</a> for data access and analysis pipelines.<br />
									<br />
									Separately, the <a href="https://usevision.org/" target="_blank"
									>VISION project</a> uses massive epigenomic datasets from hematopoetic cells lines to characterize and validate regulatory sequences in the human and mouse genomes, and to build better models of gene expression and regulation. As part of an effort to <a id="cCRE_paper_link" target="_blank">identify candidate cis-regulatory elements (cCREs) using cross-species epigenomic data</a>, I worked with fellow Ph.D. student <a href="https://kweav.github.io/" target="_blank">Kate Weaver</a> to explore the enrichment of the identified cCREs in heritability of human blood-related traits. Validating their characterization, we find that the identified cCREs are significantly enriched in and <i>highly</i> specific to human blood cell phenotypes.</p>
									<div>
										<h5>Relevant publications</h5>
									</div>
									<div class="features">
										<article style="border-top:0;margin-bottom: 1em;">
											<div class="inner" style="display: inline-block;width:83%;padding-right:2em">
												<p style="font-size:80%;line-height:1.5"><span id="MAGE_paper_longRef" /></p>
											</div>
											<span id="MAGE_paper_dimensions" style="width:8%;display:inline-block;"></span>
											<span id="MAGE_paper_altmetric" style="margin-left: 10px"></span>
											<!-- <span id="Michael test" class="altmetric-embed" style="margin-left: 10px" data-badge-popover="bottom" data-badge-type="donut" data-doi="https://doi.org/10.1093/molbev/msab258" data-hide-no-mentions="false" ></span> -->
										</article>
										<article style="border-top:0;padding-top:0">
											<div class="inner" style="display: inline-block;width:83%;padding-right:2em">
												<p style="font-size:80%;line-height:1.5"><span id="cCRE_paper_longRef" /></p>
											</div>
											<span id="cCRE_paper_dimensions" style="width:8%;display:inline-block;"></span>
											<div id="cCRE_paper_altmetric" style="display: inline-block;width:8%;vertical-align:middle;padding:0;margin:0"></div>
										</article>
									</div>

									<h4 style="padding-top:2em">Characterizing a complete human reference genome</h4>
									<a id="variants_paper_link" target="_blank" class="image" style="width:70%"><img src="images/T2T-variants-graphical-abstract_v3.png" alt="" style="width:100%"/></a>
									<p style="font-size:12px">Comparison of chr1 assembly between GRCh38 and T2T-CHM13. Adapted from <span id="variants_paper_shortRef" /></p>
									<p>The first human genome sequence—completed in 2003 as part of the Human Genome Project—covered 92% of the full genome. That last 8% (almost 200 million bases)—largely comprised of difficult-to-resolve repetitive sequences—eluded scientists for nearly two decades. However, in 2021 the <a href="https://sites.google.com/ucsc.edu/t2tworkinggroup" target="_blank">Telomere-to-Telomere (T2T) Consortium</a> finally assembled a 100% complete human genome (T2T-CHM13), using long-read technologies to fill in the missing gaps and correct errors in the previous assembly.<br />
									<br />
									As a member of this consortium, I helped characterize <a id="variants_paper_link" target="_blank">the utility of using this assembly as a reference for analysis of human genetic variation</a>. In addition to characterizing variation in previously-unresolved regions of the genome, I also facilitated the transition from the current widely used human reference genome (GRCh38) to the new T2T-CHM13 assembly by lifting over catalogs of human genetic variation (over 700 million distinct genetic variants)—to the new reference.<br />
									<br />
									Recently, the T2T Consortium finished <a id="chrY_paper_link" target="_blank">our assembly of a human Y chromosome</a>, completing the T2T-CHM13v2.0 reference genome. This assembly adds 30 million bases of new sequence (over 50% of the chromosome!) and corrects errors in the previous assembly. Building on my work with the T2T-CHM13v1.0 assembly, I led an analysis of short-read alignment and variant calling using this new assembly, characterizing the improvements offered relative to the previous reference.</p>
									<div>
										<h5>Relevant publications</h5>
										<p style="font-size: 70%; margin-bottom:1em">&#42;   denotes co-first authorship</p>
									</div>
									<div class="features">
										<article style="border-top:0;margin-bottom: 1em;">
											<div class="inner" style="display: inline-block;width:83%;padding-right:2em">
												<p style="font-size:80%;line-height:1.5"><span id="chrY_paper_longRef" /></p>
											</div>
											<span id="chrY_paper_dimensions" style="width:8%;display:inline-block;"></span>
											<div id="chrY_paper_altmetric" style="display: inline-block;width:8%;vertical-align:middle;padding:0;margin:0"></div>
										</article>
										<article style="border-top:0;padding-top:0;margin-bottom:1em">
											<div class="inner" style="display: inline-block;width:83%;padding-right:2em">
												<p style="font-size:80%;line-height:1.5"><span id="chm13structdiff_paper_longRef" /></p>
											</div>
											<span id="chm13structdiff_paper_dimensions" style="width:8%;display:inline-block;"></span>
											<div id="chm13structdiff_paper_altmetric" style="display: inline-block;width:8%;vertical-align:middle;padding:0;margin:0"></div>
										</article>
										<article style="border-top:0;padding-top:0">
											<div class="inner" style="display: inline-block;width:83%;padding-right:2em">
												<p style="font-size:80%;line-height:1.5"><span id="variants_paper_longRef" /></p>
											</div>
											<span id="variants_paper_dimensions" style="width:8%;display:inline-block;"></span>
											<div id="variants_paper_altmetric" style="display: inline-block;width:8%;vertical-align:middle;padding:0;margin:0"></div>
										</article>
										<p style="font-size: 70%;">For a full list of publications, please see my CV below.</p>
									</div>
								</div>
							</section>

						<!-- Honors and Awards -->
							<section id="awards">
								<div class="container">
									<header class="major">
										<h2>Honors and Awards</h2>
									</header>
									<h4>NIH <i>Ruth L. Kirschstein National Research Service Award/F31 Fellowship</i> (2023) </h4>
									<p>The Ruth L. Kirschstein National Research Service Award (NRSA) Individual Predoctoral Fellowship award funds promising predoctoral students, enabling them to obtain individualized, mentored research training from outstanding faculty sponsors while conducting dissertation research in scientific health-related fields relevant to the missions of the participating NIH Institutes and Centers.<br />
									<br />
									I was awarded an F31 Fellowship from the National Human Genome Research Institute (NHGRI) for my proposed work uncovering sources of human gene expression variation in a globally diverse cohort.<br />
									<span style="font-size:12px"><i>Adapted from <a href="https://grants.nih.gov/grants/guide/pa-files/pa-21-051.html" target="_blank">grants.nih.gov</a></i></span></p>
									<h4>NSF <i>Graduate Research Fellowship Program - Honorable Mention</i> (2021)</h4>
									<p>The National Science Foundation (NSF) Graduate Research Fellowship Program provides Fellowships to individuals selected early in their graduate careers based on their demonstrated potential for significant achievements in science and engineering. The NSF accords Honorable Mention to meritorious applicants who do not receive Fellowship awards. This is considered a significant national academic achievement and provides access to cyberinfrastructure resources through the Extreme Science and Engineering Discovery Environment (XSEDE).<br />
									<span style="font-size:12px"><i>Adapted from <a href="https://www.nsf.gov/pubs/2012/nsf12599/nsf12599.htm" target="_blank">nsf.gov</a></i></span></p>
									<h4>JHU Center for Teaching Excellence and Innovation <i>Instructional Enhancement Grant</i> (2021)</h4>
									<p>The Instructional Enhancement Grant Program is a mini-grant initiative that enables faculty to partner with technology savvy students to develop resources that enhance pedagogy, increase or facilitate access to course content, encourage active learning, promote critical thinking, and support student collaboration.<br />
									<span style="font-size:12px"><i>Adapted from <a href="https://ctei.jhu.edu/programs-and-services/instructional-enhancement-grant-program" target="_blank">ctei.jhu.edu</a></i></span></p>
									<h4>JHU Department of Biology Victor G. Corces Teaching Award (2021)</h4>
									<p>The Victor G. Corces Teaching Award recognizes a graduate Teaching Assistant in the Department of Biology for their outstanding efforts, enduring devotion to their students, and excellence in education in the biological sciences. This is a funded award.</p>
									<h4>Student Speaker for Univeristy of Maryland CMNS Commencement (2018)</h4>
									<p> I was selected as the Student Commencement Speaker for the University of Maryland <a href="https://cmns.umd.edu/" target="_blank">College of Computer, Mathematical, and Natural Sciences (CMNS)</a> Winter 2018 graduation ceremony.</p>
									<p style="font-size: 70%;">For a full list of awards, please see my CV below.</p>
								</div>
							</section>

						<!-- Teaching and Outreach -->
							<section id="teaching">
								<div class="container">
									<header class="major">
										<h2>Teaching</h2>
									</header>
									<h4>Quantitative Biology</h4>
									<p>Since 2020, I have served as a teaching assistant for the CMDB program's one-week <b><a href="https://bxlab.github.io/cmdb-quantbio/" target="_blank">Quantitative Biology Bootcamp</a></b>. The course is taken by all first-year Ph.D. students in the CMDB program and is designed to introduce biology students with little to no computational experience to the basics of computational biology. Students learn basic Python and Bash, and apply these languages to carry out simple analyses on biological data. <br />
									<br />
									These concepts are built on in a year-long <b><a href="https://bxlab.github.io/cmdb-quantbio/#cmdb-quantitative-biology-lab-2022" target="_blank">Quantitative Biology Lab</a></b> course, for which I also serve as a teaching assistant. Using the skills they learned in the bootcamp, students develop their own implementations of widely-used computational biology tools. These projects range from genome assembly, to sequence alignment and GWAS. <br/>
									<br />
									In addition to serving as a teaching assistant for the Quantitative Biology lab course, I also developed and teach a guest lecture for the course. <b><a href="https://bxlab.github.io/cmdb-quantbio/assignments/lab/alignment/assignment/" target="_blank">My lecture</a></b> covers local and global sequence alignment, introduces students to tools in use in the field today, and leads students in developing their own implementation of the Needleman-Wunsch dynamic programming algorithm for global sequence alignment. This lecture has been listed as one of the student's favorite lectures in anonymous reviews each year it has been taught.<br />
									<br />
									Additionally, I've served as a guest lecturer for Cold Spring Harbor Laboratory's <b><a href="https://meetings.cshl.edu/courses.aspx?course=C-SEQTEC&year=22" target="_blank">Advanced sequencing technologies and bioinformatics analysis</a></b> course, which explores the use and applications of massively parallel sequencing technologies, with a focus on data analysis and bioinformatics, including variant calling, transcriptome analysis, single cell analysis, metagenomics, and epigenomics.</p>
									<h4>Developmental Genetics</h4>
									<p>During the 2020-2021 school year, I served as a teaching assistant for the undergraduate Developmental Genetics Lab course. This course covers the basics of <i>C. elegans</i> development and introduces students to genetic modification using the CRISPR/Cas9 system. As part of the lab, students use CRISPR to add a fluorescent tag to a protein of interest, allowing them to track this protein over the course of <i>C. elegans</i> development. A large portion of this course focuses on properly designing the genetic constructs necessary to carry out their specific CRISPR modification.<br/>
									<br />
									Designing CRISPR reagents requires analysis of DNA sequences and extraction of specific strings of bases, and therefore lends itself to teaching students basic string manipulation skills. <a href="https://scarioscia.github.io/" target="_blank">Sara Carioscia</a> (a fellow TA) and I were awarded a <b><a href="https://ctei.jhu.edu/programs-and-services/instructional-enhancement-grant-program" target="_blank">Instructional Enhancement Grant Program Fellowship</a></b> by the Johns Hopkins University Center for Teaching Excellence and Innovation to design a module for the Developmental Genetics course that guides students through a computational tool for CRISPR reagent design. The module covers the basics of Python and string manipulation and then has students use these skills to build a full CRISPR reagent design tool from scratch. This module allows us to introduce computational biology to undergraduate students that may not have had previous exposure, and gives them the chance to use what they’ve learned to build a computational tool that they can use in their own research. The module, in its current state, is publicly available through <b><a href="https://github.com/dtaylo95/A-Computational-Approach-to-CRISPR-Reagent-Design" target="_blank">my GitHub</a></b>.</p>
								</div>
							</section>

						<!-- Academic Service -->
							<section id="service">
								<div class="container">
									<header class="major">
										<h2>Academic Service</h2>
									</header>
									<div class="features">
										<article>
											<a href="https://jhu.campusgroups.com/aga/home/" target="_blank" class="image"><img src="images/agara-bio.png" alt="" /></a>
											<div class="inner">
												<h5>HMM Workshop with Agara Bio</h5>
												<p>In the Fall of 2020 and Spring of 2021, I worked with Sara Carioscia to develop a computational biology workshop for Agara Bio, an undergraduate-driven community lab at JHU. We offered a <a href="https://github.com/dtaylo95/hmm_workshop" target="_blank">two-part workshop</a> covering Hidden Markov Models (HMMs), and led attendees in building their own gene-finding HMM in Jupyter Notebooks.</p>
											</div>
										</article>
										<article>
											<a href="https://cmdb.jhu.edu/" class="image" target="_blank"><img src="images/cmdb.png" alt="" /></a>
											<div class="inner">
												<h5>CMDB BioReps</h5>
												<p>The BioReps are student-elected representatives within the CMDB program that act as liasons between their graduate cohort, and the program directors and administration. We meet regularly with our cohorts, host monthly town halls, and are also involved in organizing the recruitment event for prospective students.</p>
											</div>
										</article>
									</div>
								</div>
							</section>

						<!-- Download CV -->
							<section id="cv">
								<div class="container">
									<header class="major">
										<h3>Curriculum Vitae</h3>
									</header>
									<object width="100%" height="500" type="application/pdf" data="docs/Taylor_CV_December_2023.pdf?#zoom=100&scrollbar=0&toolbar=0&navpanes=0">
										<iframe sandbox="allow-scripts allow-same-origin allow-popups" src="https://drive.google.com/file/d/11QKB7xg8q_BPi-3QMCORTvM5U1P_8r-V/preview" width="100%" height="500" frameborder="0" seamless="">
											<p>PDF cannot be displayed.</p>
										</iframe>
									</object>
									<p></br>You can download my current CV <b><a href="https://drive.google.com/uc?export=download&id=11QKB7xg8q_BPi-3QMCORTvM5U1P_8r-V">here</a></b>.<br />
									<br />
									If you'd like to contact me, feel free to reach out via twitter or email using the links provided at the bottom of the navigation pane.</p>
									</form>
								</div>
							</section>
					</div>

				<!-- Footer -->
					<section id="footer">
						<div class="container">
							<ul class="copyright">
								<li>&copy;2021 Dylan Taylor</li><li>Design: Read Only by ajlkn through <a href="http://html5up.net">HTML5 UP</a></li>
							</ul>
						</div>
					</section>

			</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.scrollex.min.js"></script>
			<script src="assets/js/jquery.scrolly.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>
			<script>
			/* 1. define variables */
			var me = "dtaylo95";
			var place = "jhu.edu";

			/* 2. find email link to replace */
			var elink = document.getElementById("mlink");

			/* 3. replace link href with variables  */
			elink.href = `mailto:${me}@${place}`;
			</script>

			<script>

				function replaceAll(id, replacedElem) {
					let elms = document.querySelectorAll(`[id='${id}']`);

					for (let i = 0; i < elms.length; i++) {
						elms[i].innerHTML = replacedElem;
					}
				}

				function getLongRef(authors, year, title, journal, doi) {
					let outString = `${authors} (${year}). <a href="https://doi.org/${doi}" target="_blank" style="color:#4acaa8">${title}</a>. <i>${journal}</i>.`;
					return outString;
				}

				function getShortRef(authors, year, journal, doi) {
					let firstAuthor_lastName = authors.split(',')[0].replace('*', '');
					let firstAuthor_initial = authors.split(',')[1];
					let journal_prefix = journal.split(',')[0];
					let outString = `<a href="https://doi.org/${doi}" target="_blank">${firstAuthor_lastName}, ${firstAuthor_initial} et al. (${year}) <i>${journal_prefix}</i>.</a>`;
					return outString;
				}

				function updateDimensions(id, doi) {
					let dimensionElem = document.getElementById(id);
					if (dimensionElem) {
						dimensionElem.setAttribute('class', "__dimensions_badge_embed__");
						dimensionElem.setAttribute('data-doi',  doi);
						dimensionElem.setAttribute('data-style', "small_circle");
						dimensionElem.setAttribute('data-legend', "hover-top");
					}
				}

				function updateAltmetric(id, doi) {
					let altmetricElem = document.getElementById(id);
					if (altmetricElem) {
						altmetricElem.setAttribute('class', "altmetric-embed");
						altmetricElem.setAttribute('data-doi',  "https://doi.org/" + doi);
						altmetricElem.setAttribute('data-badge-type', "donut");
						altmetricElem.setAttribute('data-badge-popover', "right");
						altmetricElem.setAttribute('data-hide-no-mentions', "false");
					}
				}

				function updateAllLinks(id, doi) {
					let elms = document.querySelectorAll(`[id='${id}']`);

					for (let i = 0; i < elms.length; i++) {
						elms[i].setAttribute('href', `https://doi.org/${doi}`);
					} 
				}

				$.getJSON('docs/references.json', function(data) {
					for (let i=0; i < data.length; i++) {
						
						let id = data[i].id;
						let authors = data[i].authors;
						let year = data[i].year;
						let title = data[i].title;
						let journal = data[i].journal;
						let doi = data[i].doi;
						let longRef = getLongRef(authors, year, title, journal, doi);
						let shortRef = getShortRef(authors, year, journal, doi);

						updateDimensions(id + '_dimensions', doi);
						updateAltmetric(id + '_altmetric', doi);
						updateAllLinks(id + '_link', doi);
						replaceAll(id + '_longRef', longRef);
						replaceAll(id + '_shortRef', shortRef);
					}
				})
				
			</script>
			<script defer type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>
			<script defer src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>

	</body>
</html>